deps/osxunwind.def

```diff
@@ -0,0 +1,2 @@
+-nr:___register_frame:___jl_register_frame
+-nr:___deregister_frame:___jl_deregister_frame
```

Comment:
i guess this isn't terrible, but would it perhaps be cleaner to switch to using libosxunwind via dlopen/dlsym?